scss/components/_case-overview.scss

```diff
@@ -140,22 +149,39 @@
     color: $crm-gray-matte;
   }
 
+  &:first-child,
```

Comment:
Could you describe in the PR what are the css changes that you needed to make? Cause on paper the task was as simple as implementing a vendor plugin. If you had to make changes to our own stylesheet is better to document them

Reply:
Right. But, these css changes are not related to the scroll bar and are actually the improvements in the styles.

If we see in the design, the first cell and the last cell has bottom borders short of 24px. So to make it look like that we added `:before, :after` tags and added content that looks like border with the same color as the background and superimposed them on the actual border creating the illusion that border is short of 24px from left for the first cell and short of 24px from the right of the last cell.

Reply:
> If we see in the design, the first cell and the last cell has bottom borders short of 24px. So to make it look like that we added :before, :after tags and added content that looks like border with the same color as the background and superimposed them on the actual border creating the illusion that border is short of 24px from left for the first cell and short of 24px from the right of the last cell.

sorry @swastikpareek : show, don't tell

Reply:
![c51-289- screens border issue](https://user-images.githubusercontent.com/3340537/48205936-2d080c80-e393-11e8-9094-4fd35b33d6ff.gif)